scm/driver/harness/repo.go

```diff
@@ -68,6 +68,11 @@ func (s *repositoryService) ListV2(ctx context.Context, opts scm.RepoListOptions
 	return s.List(ctx, opts.ListOptions)
 }
 
+func (s *repositoryService) ListNamespace(ctx context.Context, _ string, opts scm.ListOptions) ([]*scm.Repository, *scm.Response, error) {
+	// List already has context about namespace
```

Comment:
**minor** client has context, correcr?

Reply:
*random thought* I wonder whether we should change that at some point? at least only keep the accountid on client level?

Reply:
done, and later maybe